### PR TITLE
Add instructions for adding kube-root-ca to local system

### PR DESCRIPTION
**Pull request by lkatalin**
This enables initializing cosign to point to a TUF root running on a cluster with self-signed certs.

I also changed the `clustercert` directory to match what the fulcio patch file has for a `volumeMount`, though it does require sudo. If the patch file somehow knows where this is without this change, let me know. The cleanest way would be to abstract the path to the clustercert in the patch file but that may be more effort than we want to get into right now.

### CODE REVIEW

File: hack/configure-self-signed-cluster.md [hunk 1]
```diff
@@ -27,7 +21,7 @@ Next, run the install script as usual
 Once the fulcio-system ns exists, create the ingress-cert secret in the fulcio-system namespace
 
 ```bash
-oc create secret generic -n fulcio-system clustercert --from-file=/path/to/clustercert/tls.crt
+oc create secret generic -n fulcio-system clustercert --from-file=/var/run/clustercert/tls.crt
```
Comment (sallyom) [inline hack/configure-self-signed-cluster.md:24]: this --from-file can be anywhere, doesn't have to match the path in the container (so can be non-root location), here you pass the local path to the file, wherever it is

File: hack/configure-self-signed-cluster.md [hunk 2]
```diff
@@ -7,17 +7,11 @@ With local development clusters, often the cluster certificate is self-signed ra
 From OpenShift, the default ingress cert is in `-n openshift-ingress`
 
 ```bash
-mkdir clustercert && cd clustercert
+sudo mkdir /var/run/clustercert && cd /var/run/clustercert
```
Comment (sallyom) [inline hack/configure-self-signed-cluster.md:10]: this can stay non-root & in non-root location (as is like current dir, or, /tmp/clustercert)

**Review by sallyom — CHANGES_REQUESTED** [review]
Suggestion to move the clustercert directory to non-root location (keep it current dir or /tmp, doesn't matter)
Then, lgtm!

File: hack/configure-self-signed-cluster.md (hunk 1 above)
Comment (lkatalin) [reply hack/configure-self-signed-cluster.md:24]: Thanks! That's super interesting, so how does the container know where to look for this certificate?

Comment (sallyom) [reply hack/configure-self-signed-cluster.md]: answered in slack but I'll also put here for others :) 

`oc create secret generic -n fulcio-system clustercert --from-file=/path/on/local/clustercert/tls.crt` creates the secret.
then this is how/where the file from the secret is placed in the fulcio-server container (the patch file):
```
spec:
  template:
    spec:
      containers:
         env:
        - name: SSL_CERT_DIR
          value: /var/run/clustercert     
      volumeMounts:
        - mountPath: /var/run/clustercert
          name: clustercert
          readOnly: true
      volumes:
      - name: clustercert
        secret:
          defaultMode: 420
          items:
          - key: tls.crt
            path: tls.crt
          secretName: clustercert
```
the secret is in the fulcio-system ns - and it contains a `key: tls.crt` with `value: contents-of-tls.crt` and whatever you put as the `mountPath`, that dir will be created in the container, so there will be `/var/run/clustercert/tls.crt` will be in the fulcio container

**Review by sallyom — APPROVED** [review]
LGTM!

**Pull request merged**